### PR TITLE
Cradio v2

### PR DESCRIPTION
**Pull request by davidphilipbarr**
Fix some issues with the dtsi file causing mirror problems. 
Add readme mentioning sweep.
Update keymap with additional layer and combos.

### CODE REVIEW

**Comment by dxmh** [comment]
I flashed my config using the updated shield in this PR earlier this evening. Seems to be working perfectly so far on the [Hypergolic](https://github.com/davidphilipbarr/hypergolic) and has resolved some layout issues I had with the version currently in `main` (plus now the Bluetooth device name is nicer :ok_hand:).

(FWIW, switching to this shield from  [`cradioz`](https://github.com/davidphilipbarr/zmk-shields/tree/main/cradioz) also removed issues I was experiencing whereby the keyboard would seemly not be advertised to any host.)

Would be great for this to be merged. :+1:

File: app/boards/shields/cradio/cradio_left.conf [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# Copyright (c) 2020 The ZMK Contributors
-# SPDX-License-Identifier: MIT
+CONFIG_ZMK_SLEEP=y
```
Comment (petejohanson) [inline app/boards/shields/cradio/cradio_left.conf:1]: We'll be turning sleep on by default when ready, individual shields shouldn't be doing so yet.

File: app/boards/shields/cradio/cradio_right.conf [hunk 2]
```diff
@@ -1,3 +1 @@
-# Copyright (c) 2020 The ZMK Contributors
-# SPDX-License-Identifier: MIT
-
+CONFIG_ZMK_SLEEP=y
```
Comment (petejohanson) [inline app/boards/shields/cradio/cradio_right.conf:1]: Ditto. Also, copyright headers shouldn't be removed.

File: app/boards/shields/cradio/cradio_right.overlay [hunk 3]
```diff
@@ -3,9 +3,11 @@
  *
  * SPDX-License-Identifier: MIT
  */
- 
 #include "cradio.dtsi"
 
 &default_transform {
 	col-offset = <17>;
 };
+
```
Comment (petejohanson) [inline app/boards/shields/cradio/cradio_right.overlay:11]: Ditch the extra lines?

File: app/boards/shields/cradio/cradio_left.overlay [hunk 4]
```diff
@@ -3,5 +3,10 @@
  *
  * SPDX-License-Identifier: MIT
  */
- 
 #include "cradio.dtsi"
+
```
Comment (petejohanson) [inline app/boards/shields/cradio/cradio_left.overlay:7]: Ditch the extra lines.

File: app/boards/shields/cradio/cradio.dtsi [hunk 5]
```diff
@@ -3,48 +3,53 @@
...
-			, <&pro_micro_d 6 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
-			, <&pro_micro_d 8 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
-			, <&pro_micro_d 9 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
-			;
 	};
-	};	
 	};
+		
+		
+	&kscan0 {
```
Comment (petejohanson) [inline app/boards/shields/cradio/cradio.dtsi:33]: Why move this out like this? Just add in the initial definition like before.

File: app/boards/shields/cradio/README.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+Cradio is a firmware for a few 34 key keyboards, including Cradio and Sweep.
+
```
Comment (petejohanson) [inline app/boards/shields/cradio/README.md:2]: Can we avoid the extra lines please?

**Review by petejohanson — CHANGES_REQUESTED** [review]
A few changes needed. Thanks!

File: app/boards/shields/cradio/cradio.dtsi (hunk 5 above)
Comment (davidphilipbarr) [reply app/boards/shields/cradio/cradio.dtsi:32]: When it was in the initial definition the sides would be the same (both left, i think) moving it out again was the only way i could resolve that. This is the main issue with the current cradio shield. I'll have a look at what else could be happening. cheers.

Comment (davidphilipbarr) [reply app/boards/shields/cradio/cradio.dtsi]: this has now been resolved by a pr from dxmh!

**Review by petejohanson — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**